src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -387,25 +274,7 @@ <h4 *ngIf="displaySubsection(4, 7)">Managing Session Responses</h4>
       Here is an example of what a student might see:
     </p>
     <tm-example-box *ngIf="questionsToCollapsed[SessionsSectionQuestions.SESSION_VIEW_RESULTS]" @collapseAnim>
-      <div class="card bg-light top-padded" *ngFor="let question of exampleQuestionsWithResponses">
-        <div class="card-body" style="padding: 20px;">
-          <tm-question-text-with-info [questionNumber]="question.feedbackQuestion.questionNumber" [questionDetails]="question.feedbackQuestion.questionDetails" style="font-size: 20px;"></tm-question-text-with-info>
-          <tm-single-statistics [question]="question.feedbackQuestion.questionDetails"
-                                [responses]="question.responsesToSelf"
-                                [statistics]="question.questionStatistics"
-                                [isStudent]="true"></tm-single-statistics>
-          <tm-student-view-responses *ngIf="question.responsesToSelf.length" [responses]="question.responsesToSelf" [feedbackQuestion]="question.feedbackQuestion" [timezone]="exampleFeedbackSession.timeZone" [statistics]="question.questionStatistics"></tm-student-view-responses>
-          <div *ngFor="let responsesForOtherRecipient of question.otherResponses">
-            <tm-student-view-responses *ngIf="responsesForOtherRecipient.length" [responses]="responsesForOtherRecipient" [feedbackQuestion]="question.feedbackQuestion" [timezone]="exampleFeedbackSession.timeZone" [statistics]="question.questionStatistics"></tm-student-view-responses>
-          </div>
-          <div *ngIf="question.responsesFromSelf.length" style="margin-top: 10px;">
-            <strong>Your own responses (to others):</strong>
-            <div *ngFor="let responseFromSelf of question.responsesFromSelf">
-              <tm-student-view-responses [responses]="[responseFromSelf]" [isSelfResponses]="true" [feedbackQuestion]="question.feedbackQuestion" [timezone]="exampleFeedbackSession.timeZone" [statistics]="question.questionStatistics"></tm-student-view-responses>
-            </div>
-          </div>
-        </div>
-      </div>
+      <tm-question-response-panel [questions]="exampleQuestionsWithResponses"></tm-question-response-panel>
```

Comment:
Before:
<img width="886" alt="Screenshot 2021-11-06 at 6 25 37 PM" src="https://user-images.githubusercontent.com/7261051/140606444-22bb91be-40ac-4a42-89e9-d2736f8697f3.png">
After:
<img width="887" alt="Screenshot 2021-11-06 at 6 25 29 PM" src="https://user-images.githubusercontent.com/7261051/140606440-8f10d028-777e-4e89-83f0-a3783958084f.png">
It's good that the UI is now standardized, but we should bring back the response from "Benny Charles" to "You". Should only need a minor adjustment to the sample data.

Reply:
fixed in 5d75cbf